### ISSUE TITLE
Computed annotation to 'do not annotate terms'

### ISSUE THREAD

**Issue by krchristie** [geneontology/go-annotation#1377]
Hi

I noticed that MGI has some annotations to 'sperm part' coming from the GOC pipeline. For example, Spag17 has an GOC sourced annotation to "sperm part" by IC from "axonemal central apparatus" with the annotation extension "part_of sperm"

However, the term "sperm part" is in the "Terms not to be used for direct annotation (gocheck_do_not_annotate) set.

It seems that the GOC inference pipeline should take the do not annotate subset into account.	

thanks,

-Karen

@hdietze

**Comment by krchristie** [comment]
@pgaudet 
- I no longer see the direct annotation to the term 'sperm part' for the MGI gene mentioned in my original report (though the source annotation to "axonemal central apparatus" is still present. 
- I also do not see annotations at all DIRECTLY to the term 'sperm part' using AmiGO.

So, I no longer observe any annotations to this term. I have not checked any other "Do not annotate" terms. I'll leave it to Chris to comment on whether the implementation is appropriate to have fixed this generally.

**Comment by ValWood** [comment]
Is this covered  by the log files?

**Comment by ValWood** [comment]
@pgaudet

**Comment by suzialeksander** [comment]
@pgaudet this should be covered by [GORULE:0000008](https://github.com/geneontology/go-site/blob/master/metadata/rules/gorule-0000008.md) right?

**Comment by pgaudet** [comment]
Yes, except that this rule only gives a warning. I think we want to filter annotations under certain conditions.
Maybe for specific GOREFs?
